site/content/entry/citra-vulkan-announcement/index.md

```diff
@@ -0,0 +1,110 @@
+
+{{< sidebyside "image" ""
+    "sm3dlogl.png=FPS: 75 Speed: 125%"
+    "sm3dlvk.png=FPS: 115 Speed: 193%" >}}
+
+But oh boy, was it ever a challenge!
+
+Vulkan support is also becoming a standard feature across many emulators, such as [Dolphin](https://dolphin-emu.org/), [Ryujinx](https://ryujinx.org/), [RPCS3](https://rpcs3.net/), [yuzu](https://yuzu-emu.org/), and many others. 
+As we've mentioned before, Citra is heading into the modern age. OpenGL is considered an ageing graphics API, with buggy and slow drivers, and performance bottlenecks. A modern graphics API like Vulkan is perfect for our vision of the New™️ Citra.
+
+If you haven't read it already, we have posted a more in-detail teaser about Vulkan in Citra in our [2020 Q2 ~ 2023 Q1 Progress Report](https://citra-emu.org/entry/citra-progress-report-2023-q1/#surprise-announcement). 
+We thoroughly recommend you give that a read, as it's a really good recap of the development of Vulkan at the time of writing that.
+
+# What to expect
+
+As with anything to do with emulation, nothing is ever perfect. Things are always evolving and changing, and this is no different.
+
+## Android
+
+Attention Android users! We have not forgotten about you! We are still here, making your gaming experience on mobile even better!
+
+A very large number of Citra users currently use a phone with a Mali GPU. This isn't something specific to Citra users, the majority of phones currently on the market use a Mali GPU! Especially with Samsung phones, if you live anywhere in the world apart from the United States or China, there's a high chance you'll get one with a Mali GPU.
+
+Sadly, these GPUs tend to have rather poor OpenGL ES drivers, so Citra's performance on them generally left a lot to be desired and bordered on unplayable for most 3DS titles. Performance improved significantly for these devices after some of the initial [rasterizer cache refactor](https://github.com/citra-emu/citra/pull/6375) work on our graphics backend was done, making many less demanding titles capable of reaching 100% emulation speed. However, lots of titles were still running short of 100%. 
+
+Now, with Vulkan, these concerns can be laid to rest! Mali's vastly better Vulkan drivers allow for better, and smoother, gameplay when using Citra's Vulkan backend. You can now easily go beyond 100% emulation speed on most, if not all, games. And with the addition of asynchronous shader compilation, in-game stutter due to shaders building is greatly reduced, leaving you with a more seamless gameplay experience.
+
+{{< mp4 src="glesvk.mp4" title="Just look at the difference!" >}}
+
+If we could rewind to the terrible OpenGL ES drivers on Mali GPUs we mentioned before for a moment; another caveat of these drivers is the amount of bugs in them. This would cause graphical glitches in games, from characters being the wrong colors, to entire landscapes wigging out. 
```

Comment:
```suggestion
If we could rewind to the terrible OpenGL ES drivers on Mali GPUs we mentioned before for a moment; another caveat of these drivers is the amount of bugs in them. This would cause graphical glitches in games, from characters being the wrong colors, to entire landscapes experiencing vertex explosions. 
```
Agree with Tobi that we should nix this slang. However, it's not clear was is meant by "wigging out" - I've replaced it with vertex explosions, but if it's something else, please replace that :)

Reply:
Never heard of it being related to drugs ever, so had no idea of it's connotations. Where I'm from, it just means something is acting weirdly.